config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  get 'new_action', to: 'foreman_leapp/hosts#new_action'
+  get 'api/v2/leapp/report/:id', to: 'foreman_leapp/api/v2/report#fetch'
+  post 'api/v2/leapp/report', to: 'foreman_leapp/api/v2/report#create'
```

Comment:
Usually using namespaces (see [here](https://github.com/theforeman/foreman_remote_execution/blob/master/config/routes.rb#L46)) is preferred over directly mapping paths to controller actions